src/components/SearchForm.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, BoxProps, OutlinedTextFieldProps } from '@mui/material';
+import { useHistory } from 'react-router-dom';
+import { SearchTextField } from '../pages/search/SearchTextField';
+
+type SearchFormProps = Pick<BoxProps, 'sx'> & Pick<OutlinedTextFieldProps, 'label' | 'placeholder'>;
```

Comment:
ingen grunn  til å begrense til bare Outlined
```suggestion
type SearchFormProps = Pick<BoxProps, 'sx'> & Pick<TextFieldProps, 'label' | 'placeholder'>;
```